common/src/main/java/dev/ftb/mods/ftbquests/quest/task/ItemTask.java

```diff
@@ -103,7 +103,7 @@ public void writeData(CompoundTag nbt, HolderLookup.Provider provider) {
 	public void readData(CompoundTag nbt, HolderLookup.Provider provider) {
 		super.readData(nbt, provider);
 		// TODO: FIXME: Something here is causing items to not be read correctly
```

Comment:
This still a thing?

Reply:
No, think that was resolved a little while back. There are some issues with certain modded items (specific data components) not sync'ing well, but I think they are issues with those mods, not FTBQ.